diff --git a/sgl-kernel/pyproject.toml b/sgl-kernel/pyproject.toml
index 1b6ff11c1..8174a53dd 100644
--- a/sgl-kernel/pyproject.toml
+++ b/sgl-kernel/pyproject.toml
@@ -8,7 +8,7 @@ build-backend = "scikit_build_core.build"
 
 [project]
 name = "sglang-kernel"
-version = "0.4.4"
+version = "0.4.4+sm121a"
 authors = [
   { name="SGLang Kernel Team", email="sglang@lmsys.org" },
 ]
diff --git a/sgl-kernel/python/sgl_kernel/version.py b/sgl-kernel/python/sgl_kernel/version.py
index cd1ee63b7..3cf56ebf5 100644
--- a/sgl-kernel/python/sgl_kernel/version.py
+++ b/sgl-kernel/python/sgl_kernel/version.py
@@ -1 +1 @@
-__version__ = "0.4.4"
+__version__ = "0.4.4+sm121a"
